detox/e2e/support/ui/screen/server.ts

```diff
@@ -66,7 +66,7 @@ class ServerScreen {
                 }
             }
         }
-        await waitFor(this.usernameInput).toExist().withTimeout(timeouts.FOUR_SEC);
+        await waitFor(this.usernameInput).toExist().withTimeout(timeouts.TEN_SEC);
```

Comment:
any reason to increase to 10?

Reply:
The tests run against a /cloud instance that I am maintain. The number of channels and teams from and in CI the load varied in android emulator and in iOS simulator.

In this case, I kept a 10 sec wait before failing. Detox will keep loging every 500ms to find the element, till 10 sec and then pass or fail based on the element presence.